worker.go

```diff
@@ -210,3 +208,15 @@ func runTaskFunc(taskFunc *reflect.Value, message *TaskMessage) (*ResultMessage,
 	}
 	return getReflectionResultMessage(&res[0]), nil
 }
+
+func (w *CeleryWorker) getRegisteredQueues() []string {
+	queuesMap := map[string]interface{}{}
+	for _, v := range w.registeredTasks {
+		queuesMap[v.Queue] = nil
```

Comment:
Why are we setting this to nil here?

Reply:
I'm using the map as a set here, so the values don't matter